projects/project-player-library/src/lib/shared/subtask-card/subtask-card.component.ts

```diff
@@ -58,7 +62,9 @@ openEditSubTaskName(
   modelref.afterClosed().subscribe((res: boolean) => {
     if (res) {
       console.log('You have successfully changed the sub task name');
```

Comment:
remove all the console logs in this page

Reply:
done